lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -815,13 +824,24 @@ static void optimizeBatchNorm(Function *F) {
       // Q = W * A
       // C = b * A + B
 
-      auto filterH = cast<Variable>(CV->getFilter())->getHandle<>();
-      auto cbiasH = cast<Variable>(CV->getBias())->getHandle<>();
+      Variable *filterV = getModifiableVariable(*CV->getFilter());
```

Comment:
can you also update comment above:
```
 // Merge the Batch Normalization operation into the convolution that comes
     // before it by updating the weights of the filter.
```
It changes convolution filter and bias.

Reply:
Good catch!